frontend/src/metabase/embedding-sdk/components/SimpleDataPicker/SimpleDataPicker.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import { Icon, NavLink, Paper, ScrollArea, TextInput } from "metabase/ui";
+import type { TableId } from "metabase-types/api";
+
+interface SimpleDataPickerProps {
+  selectedEntity?: TableId;
+  options: Options[];
+  onClick: (option: any) => void;
+}
+
+interface Options {
+  id: number;
+  display_name: string;
+}
+
+const TEN_OPTIONS_HEIGHT = 10 * 33;
+
+export function SimpleDataPicker({
+  selectedEntity,
+  options,
+  onClick,
+}: SimpleDataPickerProps) {
+  const shouldShowSearchBar = options.length > 10;
+  const [searchText, setSearchText] = useState("");
+  function filterSearch(option: Options): boolean {
+    if (searchText) {
+      return normalizeString(option.display_name).includes(
+        normalizeString(searchText),
+      );
+    }
+
+    return true;
+  }
+
+  return (
+    <Paper w="300px" p="sm">
+      {shouldShowSearchBar ? (
+        <TextInput
+          data-autofocus
+          type="search"
+          icon={<Icon name="search" size={16} />}
+          mb="sm"
+          placeholder={t`Search…`}
+          onChange={e => setSearchText(e.target.value ?? "")}
+        />
+      ) : (
+        /**
+         * Behave like Mantine 7's `FocusTrap.InitialFocus`.
+         *
+         * This component disable the automatic focus on the first focusable element
+         * when there is no search bar.
+         */
+        <span aria-hidden data-autofocus tabIndex={-1} />
```

Comment:
Try to render nothing here, and see how it behaves in storybook. 
![image](https://github.com/user-attachments/assets/c26d3c5e-7305-4c1c-97f7-496b2881b89d)

The outline will look okay in the actual app, I'm not sure why it looks like this in storybook.

This is needed to ensure the same experience, as we never has any picker has the first option autofocused before (I think...)